Common/include/geometry/CGeometry.hpp

```diff
@@ -264,7 +263,7 @@ class CGeometry {
   /*!
    * \brief Constructor of the class.
    */
-  CGeometry(void);
```

Comment:
Fun Fact, there are 1475 `(void)` in SU2_CFD and Common ... We/I could make them go extinct (if we are ok with some possible minor merge conflicts in other peoples PR's )